src/index.html

```diff
@@ -18,6 +18,15 @@
                 </div>
             </nav>
         </header>
+        <main>
+            <section class="landing">
+
+            </section>
+            <section class="recipes">
+                <ul class="recipes__list">
+                </ul>
+            </section>
```

Comment:
**OPTIONAL**
Please make sure that the use of classes is reserved for a set of elements, rather than singular elements. For single elements that do not share styling consider using `id`'s.

Contrary to what the BEM convention may suggest, these elements are not repeated and must therefore have their own id.